plugins/gatsby-source-nav/gatsby-node.js

```diff
@@ -113,34 +105,27 @@ const createWhatsNewNav = async ({ createNodeId, nodeModel }) => {
     },
   });
 
-  const now = new Date();
-  const firstOfMonth = startOfMonth(now);
-  const lastMonth = sub(firstOfMonth, { months: 1 });
+  const currentYear = new Date().getFullYear();
+  const postsByYear = groupBy(posts, (post) => parseDate(post).getFullYear());
+  const thisYearsPosts = postsByYear.get(currentYear) || [];
 
-  const thisMonthsPosts = posts.filter((post) =>
-    isEqualOrAfter(parseDate(post), firstOfMonth)
+  const postsByMonth = groupBy(thisYearsPosts, (post) =>
+    parseDate(post).toLocaleString('default', { month: 'long' })
```

Comment:
TIL `toLocaleString` takes _options_. Neat!

Reply:
I learned it today as well!! Thanks Stack Overflow!